### PR TITLE
Feat: Added Login and Signup Page

### PR DESCRIPTION
**Pull request by spraveenofficial**
Added Login Page
Added Signup Page
Added Componently CDN
Added React-Helmet

### CODE REVIEW

**Review by ashishnnnnn — APPROVED** [review]
Code seems clean and Understandble ...
Can be merged..

File: src/Components/Container/index.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import "./style.css";
+const Container = ({ children }) => {
+  return (
+    <div className="container">
+      <div className="container_data">{children}</div>
+      {/* <h1>This is data</h1> */}
```
Comment (shivanigangadharan) [inline src/Components/Container/index.jsx:6]: Can remove unwanted comments

File: src/Components/Navbar/style.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  z-index: 99999999;
+  cursor: pointer;
+  position: absolute;
+  right: 0.5rem;
+  top: -11px;
+  fill: var(--white);
+}
+
+.desktop-navbar {
+  /* background: red; */
```
Comment (shivanigangadharan) [inline src/Components/Navbar/style.css:50]: here too

**Review by shivanigangadharan — APPROVED** [review]
Code looks good, nice work.

**Pull request merged**